src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorComponentSearchEngine.cs

```diff
@@ -4,23 +4,23 @@
 using System;
 using System.IO;
 using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.AspNetCore.Razor.Language.Intermediate;
-using Microsoft.AspNetCore.Razor.LanguageServer.Common;
-using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Razor;
 using Microsoft.CodeAnalysis.Razor.ProjectSystem;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer
 {
     internal class DefaultRazorComponentSearchEngine : RazorComponentSearchEngine
     {
+        private readonly ForegroundDispatcher _foregroundDispatcher;
```

Comment:
This isn't used, should be fine to remove.

Reply:
It's actually [mandatory to remove](https://dev.azure.com/dnceng/public/_build/results?buildId=748578&view=logs&j=1f132584-ab08-5fbf-e5d0-5685de454342&t=76c675cd-a096-5dd5-8431-7aa49ff610be&l=266).